### ISSUE TITLE
Perfusion PCA model requirement

### ISSUE THREAD

**Issue by ashishsingh18** [CBICA/CaPTk#1348]
Perfusion PCA GUI asks for a model directory to do PCA estimation on a batch of subjects. But there is no 'pre-trained' model provided as for other applications.

**To Reproduce**
1. Click on Applications->Perfusion PCA
2. See the dialog that opens up.

**Expected behavior**
Either a pre-trained model should be provided or if the model is not required, the GUI should be modified.

**Screenshots**
![image](https://user-images.githubusercontent.com/43445206/95116871-b35f3180-0715-11eb-936a-a5218046b25e.png)

**CaPTk Version**: 1.8.0

OS: Windows 10 x64

**Additional Info:** The documentation doesn't mention anything about the model. https://cbica.github.io/CaPTk/PCA_Extraction.html

**Comment by sarthakpati** [comment]
This is not present on my installation and not even in the master [[ref](https://github.com/CBICA/CaPTk/blob/master/src/view/gui/fMainWindow.cpp#L7603)]. This application was disabled for 1.8.0 release and is stated for release in the patch release.

**Comment by ashishsingh18** [comment]
> 
> 
> This is not present on my installation and not even in the master [[ref](https://github.com/CBICA/CaPTk/blob/master/src/view/gui/fMainWindow.cpp#L7603)]. This application was disabled for 1.8.0 release and is stated for release in the patch release.

Of-course it has been disabled now, but this was there up until the moment we disabled the application. Even now if you enable the application and open it, this will show up.

**Comment by sarthakpati** [comment]
Oh, got it, so you are opening these for you to track when you work on this application.

**Comment by ashishsingh18** [comment]
> 
> 
> Oh, got it, so you are opening these for you to track when you work on this application.

Yeah...having these on Github helps better tracking. These should have been caught during our earlier testing phases though, but better late than never..
